cmd/controller/app/controller.go

```diff
@@ -134,13 +138,13 @@ func buildControllerContext(opts *options.ControllerOptions) (*controller.Contex
 			Recorder:                        recorder,
 			KubeSharedInformerFactory:       kubeSharedInformerFactory,
 			SharedInformerFactory:           sharedInformerFactory,
-			Namespace:                       opts.Namespace,
+			Namespace:                       defaultNamespace,
 			ClusterResourceNamespace:        opts.ClusterResourceNamespace,
 			ACMEHTTP01SolverImage:           opts.ACMEHTTP01SolverImage,
 			ClusterIssuerAmbientCredentials: opts.ClusterIssuerAmbientCredentials,
 			IssuerAmbientCredentials:        opts.IssuerAmbientCredentials,
 		}),
-		Namespace:                opts.Namespace,
+		Namespace:                defaultNamespace,
```

Comment:
The namespace field can be removed from this structure altogether afaik? I don't think it's used.